make/modules/java.base/Lib.gmk

```diff
@@ -74,6 +76,7 @@ $(eval $(call SetupJdkLibrary, BUILD_LIBNIO, \
     NAME := nio, \
     OPTIMIZATION := HIGH, \
     EXTRA_HEADER_DIRS := \
+        libjava/nio/ch \
         libnio/ch \
         libnio/fs \
```

Comment:
We can't add extra header dirs on a per-platform basis, so if it is needed for AIX it will need to remain here. Otoh, the only "cost" is an additional `-I <dir>` argument to the compiler, so it's not too bad. But if we can remove it, we should, of course.

Reply:
Not changed.